library/core/src/option.rs

```diff
@@ -1166,11 +1121,9 @@ impl<T: Default> Option<T> {
     /// assert_eq!(0, bad_year);
     /// ```
     ///
-    /// [`Some`]: #variant.Some
-    /// [`None`]: #variant.None
-    /// [default value]: ../default/trait.Default.html#tymethod.default
-    /// [`parse`]: ../../std/primitive.str.html#method.parse
-    /// [`FromStr`]: ../../std/str/trait.FromStr.html
+    /// [default value]: Default::default
+    /// [`parse`]: str::parse
+    /// [`FromStr`]: crate::str::FromStr
```

Comment:
Huh, it’s interesting that this works given that previously it linked to `std`. Is that `std` link a reexport of `core`?

Reply:
Yes, a lot of std just re-exports core. https://doc.rust-lang.org/src/core/str/mod.rs.html#76-106